src/main/cpp/auton/drivePrimitives/DrivePathPlanner.cpp

```diff
@@ -96,7 +96,9 @@ void DrivePathPlanner::Init(PrimitiveParams *params)
     m_pathname = params->GetPathName(); // Grabs path name from auton xml
     m_choreoTrajectoryName = params->GetTrajectoryName();
     m_pathGainsType = params->GetPathGainsType();
+    LogStringData(DragonDataLoggerSignals::StringSignals::AUTON_PATH_NAME, m_pathname);
```

Comment:
This should be in the DataLog method

So, you will need to ...

In .h file:

void DataLog() override;

In .cpp file:

void DrivePathPlanner::DataLog()
{
    LogStringData(DragonDataLoggerSignals::StringSignals::AUTON_PATH_NAME, m_pathname);
}

Remove this line here